CorsixTH/Lua/cheats.lua

```diff
@@ -83,7 +83,10 @@ function Cheats:cheatResearch()
 end
 
 function Cheats:cheatEmergency()
-  if not self.hospital:createEmergency() then
+  local outcome, err = self.hospital:createEmergency()
+  if err == "undiscovered_disease" then
+    self.hospital.world.ui:addWindow(UIInformation(self.hospital.world.ui,{_S.misc.cant_treat_emergency}))
```

Comment:
If you make a local `ui` reference, that saves many `self.hospital.world.ui` de-references, and it makes the lines of code shorter as well.

Reply:
This one is getting fixed in next revision